docs/services/webhostingsolutions.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Web hosting solutions at OCF"
```

Comment:
Can the title be improved? Maybe just "Web hosting solutions"

Reply:
Sure, that works. I wanted to emphasize that this is the overview page while the others are the specific pages for each type.

Reply:
Could we title it "What type of web hosting do I need?" Of course, I realize this isn't necessarily consistent with the rest of the documentation since none of the other pages are titled with questions.

Reply:
I think the break in consistency could be helpful in this context since it would grab the user's attention